src/script/components/Conversation/Conversation.tsx

```diff
@@ -566,3 +596,26 @@ export const Conversation = ({
     </DropFileArea>
   );
 };
+
+interface JumpToLastMessageButtonProps extends HTMLProps<HTMLElement> {
+  onGoToLastMessage: () => void;
+  conversation: ConversationEntity;
+}
+
+export const JumpToLastMessageButton: FC<JumpToLastMessageButtonProps> = ({
+  onGoToLastMessage,
+  conversation,
+  ...rest
+}: JumpToLastMessageButtonProps) => {
+  const {isLastMessageVisible} = useKoSubscribableChildren(conversation, ['isLastMessageVisible']);
+
+  if (isLastMessageVisible) {
+    return null;
+  }
+
+  return (
+    <IconButton data-uie-name="jump-to-last-message-button" onClick={onGoToLastMessage}>
+      <ChevronIcon css={{rotate: '90deg', height: 16, width: 16, path: {fill: '#0667C8'}}} />
+    </IconButton>
+  );
+};
```

Comment:
Let's extract that to its own file

Reply:
Initially I did so, but then I saw we have such small components in the same file, will do so